content/docs/sidebar.json

```
@@ -102,7 +102,12 @@
           "katacoda": "https://katacoda.com/dvc/courses/examples/dvcignore"
         }
       },
-      "updating-tracked-files",
+      {
+        "label": "How To",
+        "slug": "how-to",
+        "source": false,
+        "children": ["update-tracked-files"]
```

To show the Best Practices how to, we need:
```suggestion
        "children": ["best-practices", "update-tracked-files"]
```